compiler/test/runnable/safearrgrow.d

```diff
@@ -0,0 +1,16 @@
+// REQUIRED_ARGS:  -d
+struct S
+{
+    string[] a;
+    @safe void reserve() scope
+    {
+        a.length += 1;
+    }
+}
+
+void main()
+{
+    S s;
+    s.reserve;
+    assert(s.a.length >= 1);
```

Comment:
This test seems unrelated to the issue. I'd make this a compilable test, runnable tests are slower.